### PR TITLE
added ilo5, updated ilo2, removed double ilo196

### PR DESCRIPTION
**Pull request by Firminator**

### CODE REVIEW

File: firmware.conf [hunk 1]
```diff
@@ -1,8 +1,3 @@
-[ilo]
-version = 1.96
-url = https://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p1980791503/v97349/CP023365.scexe
-file = ilo196.bin
-
```
Comment (seveas) [inline firmware.conf]: This is not a duplicate section, but a definition of the 'default' firmware for the ilo1. ilo2/3/4 have a similar section

File: firmware.conf [hunk 2]
```diff
@@ -502,3 +502,9 @@ file = ilo4_254.bin
 version = 2.55
 url = https://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p192122427/v129421/CP032487.scexe
 file = ilo4_255.bin
+
+[ilo5 1.20]
+version = 1.20
+url = http://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p1342933511/v138105/CP032773.scexe
+file = ilo5_120.bin
+
```
Comment (seveas) [inline firmware.conf]: Could you add a version-less ilo5 section too please?

File: firmware.conf [hunk 3]
```diff
@@ -258,6 +253,11 @@ version = 2.32
 url = https://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p915269625/v144949/CP034429.scexe
 file = ilo2_232.bin
 
+[ilo2 2.33]
+version = 2.33
+url = https://downloads.hpe.com/pub/softlib2/software1/sc-linux-fw-ilo/p915269625/v148520/CP035238.scexe
+file = ilo2_233.bin
+
```
Comment (seveas) [inline firmware.conf]: This is already part of #176

**Comment by Firminator** [comment]
Closing and redoing it with just adding ilo5 v1.20
Thanks.

**Pull request closed without merging**